prow/config/cache.go

```diff
@@ -380,8 +380,9 @@ func (cache *InRepoConfigCache) GetInRepoConfig(identifier string, baseSHAGetter
 // because unit tests can just provide its own function for constructing a
 // *ProwYAML object (instead of needing to create an actual Git repo, etc.).
 func (cache *InRepoConfigCache) getProwYAML(
-	valConstructorHelper func(git.ClientFactory, string, RefGetter, ...RefGetter) (*ProwYAML, error),
+	valConstructorHelper func(git.ClientFactory, string, string, RefGetter, ...RefGetter) (*ProwYAML, error),
```

Comment:
It might improve readability to define this func type once somewhere, with named fields, or add names here. (As multiple "strings" now occur.)

Reply:
> It might improve readability

If you scroll down a bit inside this function (I guess GitHub's default diff context size did not cover this), you'll see that it is invoked immediately in with all the correct arguments (already aptly named).

> define this func type once somewhere, with named fields, or add names here.

Out of these options, which one do you think is the most most readable? Also I can't tell if your comment is "please fix" or "this is a nit; feel free to ignore".

Reply:
It's not required.

Reply:
Oh, in answer, the most readable is probably the type defined above the function. It's a bit unwieldily to read without the extra length of names in side-by-side diffs. Note it could be reused below (in tests, iirc).

Completely up to you.

Reply:
> the most readable is probably the type defined above the function.

Ack. I'll do this in a follow-up PR.